examples/fluent-bit/logstash/README.md

```diff
@@ -0,0 +1,43 @@
+### FireLens Example: Sending Logs to hosted Logstash with Fluent Bit
+
+You can use Fluent Bit's [HTTP output plugin](https://docs.fluentbit.io/manual/pipeline/outputs/http) to send your container's log to external or aws hosted logstash. 
```

Comment:
wait- AWS has hosted logstash?? Or do you mean logstash running on AWS (in EC2)?

Reply:
oh I see, its part of amazon elasticsearch, nvm